src/widgets/about-school/ui/about-school.module.scss

```diff
@@ -0,0 +1,3 @@
+.about-school {
+  min-width: 300px;
+}
```

Comment:
Do we need this styles at all?

Reply:
Actually we don't but we can't use empty `about-school.module.scss` file and can't delete it because we have to use `scss-modules`

Reply:
we're using scss-modules instead of plain css. If there is no styles for component why fake it?

Reply:
removed it